parm/postxconfig-NT-fv3lam_rrfs.txt

```diff
@@ -9712,19 +9971,19 @@ spec_hgt_lvl_above_grnd
 ?
 ?
 ?
-791
-GSD_MIN_UPHL_ON_SPEC_HGT_LVL_ABOVE_GRND_0-3km
+788
+GSD_MAX_UPHL_ON_SPEC_HGT_LVL_ABOVE_GRND_0-2km
 ?
 1
 tmpl4_8
-MNUPHL
+MXUPHL
```

Comment:
@AndrewBenjamin-NOAA This field was generated from your test. Can you remove it?

Reply:
@WenMeng-NOAA I removed MXUPHL and MNUPHL from 0-2km. You have MNUPHL highlighted here from 0-3km, which is not part of my changes in this PR. The MNUPHL and MXUPHL that is on the parallels looks good, so they should stay.